src/Compilers/CSharp/Portable/CodeGen/CodeGenerator.cs

```diff
@@ -348,6 +348,11 @@ private void EmitSymbolToken(FieldSymbol symbol, SyntaxNode syntaxNode)
             _builder.EmitToken(_module.Translate(symbol, syntaxNode, _diagnostics), syntaxNode, _diagnostics);
         }
 
+        private void EmitSymbolToken(FunctionPointerTypeSymbol symbol, SyntaxNode syntaxNode)
```

Comment:
>private void EmitSymbolToken(FunctionPointerTypeSymbol symbol, SyntaxNode syntaxNode) [](start = 8, length = 85)

It looks like if we call ```private void EmitSymbolToken(TypeSymbol symbol, SyntaxNode syntaxNode)``` with the same arguments, we get something different. Consider changing the name of this method to make it clear that it is not an overload.